tools/cjxl_ng_main.cc

```diff
@@ -277,60 +244,76 @@ DEFINE_string(frame_indexing, "",
               "the frame index box.");
 
 DEFINE_string(
-    // TODO(tfish): Clarify with team whether changing from int-param to string
-    // is OK here.
     colortransform, "",
     "The color transform to use. Valid values are: '' (= \"use default\"), "
     "'RGB', 'XYB', 'YCbCr'.");
```

Comment:
It's probably a good idea to mention here that `YCbCr` actually means that the input will be interpreted as if it is in YCbCr already, i.e. we're not doing the forward color transform (RGB to YCbCr) but just encoding the pixel values as-is and signaling that they are YCbCr. This is surprising and is only useful for lossless compression of already-YCbCr data (like video frames in .y4m) that also happens to be 4:4:4, since we don't currently have a way to pass chroma subsampled data using the encode api.

Reply:
Changed docstring. Does this address the issue?